File: golang/round267/467A.go
package main

import "fmt"

func main() {
	var n int
	fmt.Scan(&n)
	var p, q int
	res := 0
	for i := 0;i < n; i++ {
		fmt.Scan(&p, &q)
		if q - p >= 2 {
			res += 1
		}
	}
	fmt.Println(res)
}

File: python/round267/467A.py
from sys import stdin

n, m, k = list(map(int, stdin.readline().split()))
a = []
for _ in range(m+1):
    a.append(int(stdin.readline()))
res = 0
for i in a[:-1]:
    t = bin(i ^ a[-1])
    count = 0
    for c in t:
        if c == '1':
            count += 1
    if count <= k:
        res += 1
print(res)
